README.md

```
@@ -51,6 +51,11 @@ You can run the OPML Manager using the following commands:
   cargo run --release -- report <input_file> <output_file> --validate-feeds --timeout <timeout_in_seconds>
   ```
 
+- **Launch the interactive TUI:**
+  ```bash
```

Markdownlint (0.37.0)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)